src/rpp/rpp/schedulers/details/queue_worker_state.hpp

```diff
@@ -74,7 +74,7 @@ class queue_worker_state
         return false;
     }
 
-    void reset()
+    void reset() noexcept
```

Comment:
Don't overcomplicate and avoid trying to do anything as noexcept =) This function is pretty simple and can be inlined by compiler without any tips